failed_rules.md

```diff
@@ -0,0 +1,3 @@
+## Failed rule proposals
```

Comment:
Shouldn't this file be called log or history or something? Since it will include both failed rules and repealed rules?

Reply:
history is fine